src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -13442,6 +13442,39 @@ class C
             Directory.Delete(dir.Path, true);
         }
 
+        [Theory]
+        [InlineData("subdir")]
+        [InlineData("a/b/c")]
+        [InlineData(" subdir")]
+        [InlineData(" a/ b/ c")]
```

Comment:
>[InlineData(" a/ b/ c")] [](http://example.com/codeflow?start=8&length=24)

Consider testing:
`"abc/"`
`"abc/ "`

Reply:
`"abc/ "` fails because it results in file name `"abc/ /generatedSource.cs"` which contains invalid segment `"/ /"`